client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -52,7 +52,7 @@ const changeCommentStatus = ( { dispatch, getState }, action ) => {
 			},
 			{
 				...action,
-				previousStatus,
+				meta: Object.assign( {}, action.meta, { comment: { previousStatus } } ),
```

Comment:
Any reason to not spread here?
`meta: { ...action.meta, comment: { previousStatus } }`

Reply:
action.meta can be `null`